uiowa/uiowa
# Fix filter_iframe schema

Relates to #8218
 




# How to test

On `main` try to save the admin/config/content/formats/manage/filtered_html configuration. You will get multiple errors, but one of them will be about allowed_sources not being a valid key.

Go to admin/reports/config-inspector/filter.format.filtered_html/list and see that filters.filter_iframe.settings.allowed_sources has some undefined columns and errors.

On `fixing_text_formats` re-sync a website and try saving the admin/config/content/formats/manage/filtered_html configuration again. You might still get multiple errors, but the allowed_sources one won't be one of them.

Go to admin/reports/config-inspector/filter.format.filtered_html/list and see that filters.filter_iframe.settings.allowed_sources no longer has issues.
